apps/widget-configurator/src/app/configurator/consts.ts

```diff
@@ -1,3 +1,57 @@
-import { TradeType } from '@cowprotocol/widget-lib'
+import { TradeType, TokenList } from '@cowprotocol/widget-lib'
 
 export const TRADE_MODES = [TradeType.SWAP, TradeType.LIMIT, TradeType.ADVANCED]
+
+// Sourced from https://tokenlists.org/
+export const TOKEN_LISTS: TokenList[] = [
+  { name: 'CoW Protocol', url: 'https://files.cow.fi/tokens/CowSwap.json' },
+  { name: '1inch', url: 'https://tokens.1inch.eth.link' },
+  { name: 'Aave Token List', url: 'https://tokenlist.aave.eth.link' },
+  { name: 'Agora dataFi Tokens', url: 'https://datafi.theagora.eth.link' },
+  {
+    name: 'BA ERC20 SEC Action',
+    url: 'https://raw.githubusercontent.com/The-Blockchain-Association/sec-notice-list/master/ba-sec-list.json',
+  },
```

Comment:
This list probably shouldn't be here

Reply:
Why not?

Reply:
Do we want to have Uniswap's token block list as a list of tokens integrators want to enable in their UI?

Reply:
Fixed